onnx/defs/experiments/defs.cc

```diff
@@ -212,6 +207,11 @@ ONNX_OPERATOR_SCHEMA(GivenTensorFill, ONNX_DOMAIN, 1, OpSchema()
         }
     }));
 
+const char * Scale_ver1_doc = R"DOC(
```

Comment:
What's the reason of moving this string outside? This makes the schema (in c++ files) less readable.

Reply:
This is the workaround I mentioned for a GCC bug.  Raw strings in macros were busted.
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=57824